controller/volume_controller.go

```diff
@@ -888,10 +896,47 @@ func isFirstAttachment(rs map[string]*longhorn.Replica) bool {
 	return true
 }
 
+func isHealthyAndActiveReplica(r *longhorn.Replica) bool {
+	if r.Spec.FailedAt != "" {
+		return false
+	}
+	if r.Spec.HealthyAt == "" {
+		return false
+	}
+	if !r.Spec.Active {
+		return false
+	}
+	return true
+}
+
+func isDefinitelyHealthyAndActiveReplica(r *longhorn.Replica) bool {
```

Comment:
Perhaps the difference between this and the previous method is clearer if we name it something like "isUsableAsLastReplica(r)"  Or we can just that in a function comment.